universal-login-contracts/test/argumentsEncoding.ts

```diff
@@ -24,15 +21,17 @@ describe('UNIT: argumentsEncoding', async () => {
     let signature2: string;
 
     before(async () => {
-      signature1 = await messageSignature(wallet1, wallet1.address, wallet1.address, value, data, nonce, gasToken, gasPrice, gasLimitExecution, gasData);
-      signature2 = await messageSignature(wallet1, wallet1.address, wallet2.address, value, data, nonce, gasToken, gasPrice, gasLimitExecution, gasData);
+      signature1 = await messageSignature(
+        wallet1, wallet1.address, wallet1.address, value, data, nonce, ETHER_NATIVE_TOKEN.address, DEFAULT_GAS_PRICE, DEFAULT_GAS_LIMIT_EXECUTION, gasData);
```

Comment:
```ts
const gasToken = ETHER_NATIVE_TOKEN.address;
const gasPrice = DEFAULT_GAS_PRICE;
const gasLimitExecution = DEFAULT_GAS_LIMIT_EXECUTION;
```

It is more readable and requires fewer changes. It concerns the following tests either.

Reply:
Done